documentation/blog/2024-09-01-whats-new-in-svelte-september-2024.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "What's new in Svelte: September 2024"
+description: 'More useful `svelte:options` and `:global`, plus the deprecation of `<svelte:component>` in Svelte 5'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+A bunch of updates to Svelte dropped in the last few weeks, plus a few quality of life improvements in the Svelte language tools.
+
+Let's take a look...
+
+## What's new in Svelte and Language Tools
+
+- `$state.frozen` has been replaced with `$state.raw` (**5.0.0-next.218**, [Docs](https://svelte-5-preview.vercel.app/docs/runes#state-raw), [#12808](https://github.com/sveltejs/svelte/pull/12808))
+- `$state.is` has been removed. RIP, little guy ([#12916](https://github.com/sveltejs/svelte/pull/12916))
+- `$state.link` was added, experimented with, and then removed (**5.0.0-next.229 - 232**, [#12545](https://github.com/sveltejs/svelte/pull/12545) and [#12943](https://github.com/sveltejs/svelte/pull/12943))
```

Comment:
I'd like to remove this, it was only there for a few hours and mentioning it prominently can only add confusion
```suggestion
```